CMakeLists.txt

```diff
@@ -55,6 +55,7 @@ include(${CMAKE_SOURCE_DIR}/cmake/FindOpenMP.cmake)
 if (METIS_LIBRARY)
   include(${CMAKE_SOURCE_DIR}/cmake/FindMetis.cmake)
 endif()
+include(${CMAKE_SOURCE_DIR}/cmake/FindFAISS.cmake)
```

Comment:
Have you tested if FAISS was not found, does the cmake still build properly?

Reply:
Yes, I tested it. Also Jenkins tested it. There is no FAISS on daisy.